README.md

```diff
@@ -52,13 +52,13 @@ Follow the instructions [here](https://wave.h2o.ai/docs/installation) to downloa
 
 
 
+## Note
+While trying to run any of the apps particularly on Windows, below are given some common errors and their fixes:
 
+**1. Makefile issue:** This is due to the os confusing between 'bin' and 'Scripts'. If you face this issue, open the Makefile present in the app folder using some Text Editor, and replace the word 'bin' with 'Scripts' in the virtual environment path of Setup and Run sections.
 
+**2. make command not found:** This error comes when make is not installed on your OS. You can install make easily in Windows. More info on installing make is available on the Internet.
 
-
-
-
-
-
+**3. Python not found:** This error is probably if Windows is not able to recognise the path of Python. The path of Python can be updated in the pyvenv.cfg file, located in [App Folder]/venv
```

Comment:
yes , this error is shown atmost beginning and nothing works in that case, as venv not works.

Reply:
> as venv not works

Then `The path of Python can be updated in the pyvenv.cfg file, located in [App Folder]/venv` is an invalid statement, right?

Reply:
Actually , In the app folder there is a pre loaded folder venv which comes when we clone the repo. 
Now in this venv folder ,there is pyvenv.cfg file. So this file and the venv folder comes with the app. So if python is not found, it is because in this preloaded file, the address is wrong
And the Make file takes the path of python from here i guess.
So i believe this is valid statement. please do update me if anything wrong.

Reply:
I have committed and specified this thing in the file to avoid confusion. :)

Reply:
> In the app folder there is a pre loaded folder venv which comes when we clone the repo

I have looked through the repo, but could not find the `venv` as you say. Can you point out where you see it?

![image](https://user-images.githubusercontent.com/64769322/196109647-94642b2f-99f2-407b-8dfb-cce460ea108a.png)

Reply:
Oh thanks for highlighting that. Yes it is not there. I guess that error was specific to my pc due to difference in directories of python installation and wave-apps. 
So I guess it will be good to remove this

Reply:
The missing python installation is a valid error and should be documented. You correctly included it, the only problem was your problem resolution. Please put it back with the proper way for fixing it - installing python, putting it to the `PATH` etc.

Reply:
As requested, I have added the python error. Kindly check